it/uk/gov/hmrc/soletraderidentificationfrontend/controllers/PersonalDetailsControllerISpec.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.soletraderidentificationfrontend.controllers
+
+import play.api.libs.ws.WSResponse
+import play.api.test.Helpers.OK
+import uk.gov.hmrc.soletraderidentificationfrontend.utils.ComponentSpecHelper
+
+class PersonalDetailsControllerISpec extends ComponentSpecHelper {
+
+  "GET /personal-details-page" should {
```

Comment:
It'd be good to discuss as a team what we'd like to be included in the component test. We had some success with doing view tests within here before.